tools/peakachu/peakachu.xml

```diff
@@ -114,22 +125,26 @@
                     <option value="gtest" selected="True">gtest</option>
                     <option value="deseq">DESeq2</option>
                 </param>
-                <param name="norm_method" type="select" label="Normalisation Method.">
-                    <option value="tmm" selected="True">TMM</option>
-                    <option value="deseq">DESeq2</option>
-                    <option value="count">Count</option>
-                    <option value="manual">Manual</option>
-                    <option value="none">None</option>
-                </param>
+                <conditional name="norm_method">
+                    <param name="norm_method_selector" type="select" label="Normalisation Method.">
+                        <option value="tmm" selected="True">TMM</option>
+                        <option value="deseq">DESeq2</option>
+                        <option value="count">Count</option>
+                        <option value="manual">Manual</option>
+                        <option value="none">None</option>
+                    </param>
+                    <when value="manual">
```

Comment:
and here as well, what happens in the other cases?
A <when/> is at least needed

Reply:
Then everything is done internally by Peakachu, you don't need to provide your own normalization factors

Reply:
when it's `None` it won't normalize the counts ... so no additional info is needed

Reply:
Then please include the closing when.

Reply:
I'm not sure I understand ... see line 142 ?